src/application-insights/azext_metadata.json

```diff
@@ -0,0 +1,3 @@
+{
+    "azext.isPreview": true
+}
```

Comment:
Your whl file is not in the `azext_applicationinsights` namespace as your setup.py indicates. Please move it.